src/system/process.py

```diff
@@ -78,10 +80,14 @@ def pid(self) -> int:
 
     @property
     def stdout_data(self) -> Any:
+        if self.started:
+            self.stdout.seek(0)
```

Comment:
This will break `if not self.stdout`.

Reply:
seek(0) can only be called before we terminate the process. So this throws an error if we try to access stdout_data after terminating the process.So I tried to return an empty string in that case.

Reply:
If there is already data in stdout, and you do not add `seek(0)`, what would be the outcome of it?

Reply:
I see an empty string while reading stdout or stderr, if I do not add seek(0).